epoxy-processor/src/main/java/com/airbnb/epoxy/processor/Memoizer.kt

```diff
@@ -359,6 +359,17 @@ class Memoizer(
             }
         }
     }
+
+    private val implementsModelCollectorMap = mutableMapOf<GeneratedModelInfo, Boolean>()
+    fun implementsModelCollector(modelInfo: GeneratedModelInfo): Boolean {
+        return synchronized(typeMap) {
+            implementsModelCollectorMap.getOrPut(modelInfo) {
+                modelInfo.superClassElement.interfaces.any {
```

Comment:
Should this check the super class recursively? If there are multiple subclasses a superclass up the chain could have the interface instead.

maybe `interfaces.any {...}  || implementsModelCollector(superClassElement.superclass)` (psuedocode)